scripts/beachball/getVNextChangelogGroups.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangelogGroupOptions } from 'beachball';
+import { AllPackageInfo, getAllPackageInfo } from '../monorepo/index';
+
+/**
+ * Generates grouped changelog for vNext
+ * @returns Grouped changelog configuration
+ */
+export function getVNextChangelogGroups(): ChangelogGroupOptions {
+  const allPackageInfo = getAllPackageInfo();
+  const vNextPackagePaths = getVNextPackagePaths(allPackageInfo);
+
+  return {
+    masterPackageName: '@fluentui/react-components',
+    changelogPath: 'packages/react-components',
+    include: vNextPackagePaths,
+  };
+}
+
+export function getVNextPackagePaths(allPackageInfo: AllPackageInfo) {
+  return Object.values(allPackageInfo)
+    .map(packageInfo => {
+      if (packageInfo.packageJson.version.startsWith('9.')) {
```

Comment:
I just added this thing which might be useful https://github.com/microsoft/fluentui/blob/master/scripts/monorepo/isConvergedPackage.js

Reply:
It's a good callout, I'll use it in this PR

Reply:
I've made some modifications to the util to improve the typing